modules/ROOT/pages/tokenization.adoc

```diff
@@ -24,6 +27,16 @@ A connection is established between the API gateway and the tokenization service
 
 The API gateway replaces the credit card number with the tokenized data and the request is sent to the app. 
 
+== Masking
+
+Anypoint Security also provides masking which is another tokenization option that can be used to hide or mask identity. When using masking, a configurable mask character is returned which hides the identity of the sensitive data. 
```

Comment:
```suggestion
Anypoint Security also provides _masking_, which is another tokenization option that can be used to hide or mask identity. When using masking, a configurable mask character is returned which hides the identity of the sensitive data. 
```

Reply:
"is returned" is passive - any way to make active?

Reply:
This sentence activized by me.